docs/scatacseq/README.md

```diff
@@ -144,7 +144,7 @@ The specific type of assay being executed.
 
 | constraint | value |
 | --- | --- |
-| enum | `SNARE-seq2`, `scATACseq`, `sciATACseq`, or `snATACseq` |
+| enum | `scRNA-seq (SNARE-seq) [Salmon]`, `scATAC-seq (SNARE-seq) [SnapATAC]`, `scRNA-seq (SNARE-seq) [Lab Processed]`, `scATAC-seq (SNARE-seq) [Lab Processed]`, `sciATAC-seq`, `sciATAC-seq [SnapATAC]`, `snATAC-seq`, or `snATAC-seq [SnapATAC]` |
```

Comment:
Please make sure this list is accurate: nothing extra, and nothing missing. ... I'm really just going on string similarity

Reply:
@mccalluc 
"sn" means single cell whereas "SN" in SNARE means single nucleus so these are 2 different assay types and should not be combined.

Reply:
2 `sn` items removed

> Salmon & Lab Processed are not assay types

They are types in assay_types.yaml, so in some sense they are? Maybe talk with Phil to resolve this?

Reply:
Fixed in https://github.com/hubmapconsortium/search-api/pull/185